huxley/www/js/components/ChairSummaryView.js

```diff
@@ -0,0 +1,185 @@
+/**
+* Copyright (c) 2011-2016 Berkeley Model United Nations. All rights reserved.
+* Use of this source code is governed by a BSD License (see LICENSE).
++*/
+
+'use strict';
+
+var React = require('react');
+var ReactRouter = require('react-router');
+
+var Button = require('components/Button');
+var AssignmentStore = require('stores/AssignmentStore');
+var CountryStore = require('stores/CountryStore');
+var CurrentUserStore = require('stores/CurrentUserStore');
+var DelegateActions = require('actions/DelegateActions');
+var DelegateStore = require('stores/DelegateStore');
+var InnerView = require('components/InnerView');
+var User = require('utils/User');
+
+var ChairSummaryView = React.createClass({
+  mixins: [
+    ReactRouter.History,
+  ],
+
+  getInitialState() {
+    var user = CurrentUserStore.getCurrentUser();
+    return {
+      assignments: AssignmentStore.getCommitteeAssignments(user.committee),
+      countries: CountryStore.getCountries(),
+      country_assignments: {},
+      delegates: DelegateStore.getCommitteeDelegates(user.committee),
+    };
+  },
+
+  componentWillMount() {
+    var user = CurrentUserStore.getCurrentUser();
+    if (!User.isChair(user)) {
```

Comment:
Do we have to check this here? It seems like there should be a more central place where we check if a user has permission to view a particular page. Did we never build that in? Can chairs go to `advisor/whatever` and see that view (even though it'll be broken)?